lib/icingadb/redisconnection.cpp

```diff
@@ -577,6 +572,7 @@ void RedisConnection::WriteItem(boost::asio::yield_context& yc, RedisConnection:
 		}
 
 		m_QueuedReads.Set();
+		m_NoQueuedReads.Clear();
```

Comment:
Why not perform this within the `ReadLoop` method once `m_QueuedReads.Wait()` returns?